extensions/amp-base-carousel/0.1/child-layout-manager.js

```diff
@@ -261,7 +267,19 @@ export class ChildLayoutManager {
         return isIntersecting;
       })
       .forEach((entry) => {
-        const {target} = entry;
```

Comment:
Seems to me like either we should:
a. make a comment explaining that this if condition means it isn't a slide but actually an AMP element within that slide and therefore has a completely separate flow
b. since there is so little overlap, seems like it could be its own InOb with a different callback

WDYT?

Reply:
I'll add a comment here. I want to keep it in the same InOb, since after we process entries, we flush out the changes all at once. I don't want to be in a situation where the same entries are processed by 2 InOb's which might schedule the layout for the slide twice.